<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemberDashboard</class>
 <widget class="QWidget" name="MemberDashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelMemberName">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Member name</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelWelcomeMember">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>601</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
 background-color:rgb(53,20,68);
color:white
}</string>
   </property>
   <property name="text">
    <string>                Welcome, </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLogOut">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>380</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Log out</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBorrowedBooks">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>251</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Borrowed Books </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBorrowReturnBook">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Borrow/Return a Book</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOverdueBooks">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>120</y>
     <width>201</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Overdue Books</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelTotalBooks">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Total Books </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <zorder>labelWelcomeMember</zorder>
  <zorder>labelMemberName</zorder>
  <zorder>pushButtonLogOut</zorder>
  <zorder>labelBorrowedBooks</zorder>
  <zorder>pushButtonBorrowReturnBook</zorder>
  <zorder>labelOverdueBooks</zorder>
  <zorder>labelTotalBooks</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
